repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -52,8 +52,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
-import static org.apache.atlas.repository.Constants.TASK_GUID;
-import static org.apache.atlas.repository.Constants.TASK_STATUS;
+import static org.apache.atlas.repository.Constants.*;
```

Comment:
Don't import constants like this. Specify exactly what you need.

Reply:
Done